### PR TITLE
Fix Issue 113- Multiple Notifications w/ CTA Click

### PR DESCRIPTION
**Pull request by FrancescoSTL**
It is currently 12:05 AM and I'm pretty tired but I'm fairly certain this is the best way to solve the issue 🇺🇸 💻 

Issue ref: #113 

r? @schalkneethling

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -433,8 +437,9 @@ function showSidebar(sidebarProps) {
 
             // listen for events when a user completes a sidebar cta
             worker.port.on('cta_complete', function() {
-                // assign new token and notify sidebar
-                assignTokens(sidebarProps.step, worker);
+                // assign new token and notify sidebar as long as we haven't done so already
+                if(!assignedToken)
+                    assignTokens(sidebarProps.step, worker);
```
Comment (schalkneethling) [inline index.js]: Nit: Missing opening and closing curly braces

**Comment by schalkneethling** [comment]
One small nit r+ w/c

**Comment by FrancescoSTL** [comment]
@schalkneethling 🇬🇧🇬🇧 thats a wrap, folks. r?

**Comment by schalkneethling** [comment]
r+ 🐒

**Pull request merged**